app/Config/Logger.php

```diff
@@ -1,133 +1,186 @@
-	|   4 = Runtime Errors      - Don't need immediate action, but should be monitored.
-	|   5 = Warnings               - Exceptional occurrences that are not errors.
-	|   6 = Notices            - Normal but significant events.
-	|   7 = Info             - Interesting events, like user logging in, etc.
-	|   8 = Debug                - Detailed debug information.
-	|   9 = All Messages
-	|
-	| You can also pass an array with threshold levels to show individual error types
-	|
-	| 	array(1, 2, 3, 8) = Emergency, Alert, Critical, and Debug messages
-	|
-	| For a live site you'll usually enable Critical or higher (3) to be logged otherwise
-	| your log files will fill up very fast.
-	|
-	*/
-	public $threshold = 3;
+	// *******************************************************************************
+	//                           File Logging Settings
+	// *******************************************************************************
 
-	/*
-	|--------------------------------------------------------------------------
-	| Error Logging Directory Path
-	|--------------------------------------------------------------------------
-	| By default, logs are written to WRITEPATH . 'logs/'
-	| Specify a different destination here, if desired.
-	*/
-	public $path = '';
+	/**
+	 * The $fileLevelsHandled property defines the message severity levels that
+	 * should be written to the log file.
+	 *
+	 * Here are the values and the message type
+	 *     0 - off
+	 *     1 - 'emergency'
+	 *     2 - 'alert'
+	 *     3 - 'critical'
+	 *     4 - 'error'
+	 *     5 - 'warning'
+	 *     6 - 'notice'
+	 *     7 - 'info'
+	 *     8 - 'debug'
+	 *
+	 * Setting a value of 0 (zero) turns the file logging off.
+	 * You can enable logging by setting a value in the range 1 to 8.
+	 * Not setting a value (leaving it blank) will cause a LogException with
+	 * the message - 'null' is an invalid log level.
+	 *
+	 * If a single value is supplied all severity levels less than or equal to the value
+	 * will be logged. In other words, setting
+	 *     $levelsHandled = 3;
+	 * would mean that critical, alert, and emergency messages would be logged.
+	 *
+	 * You can also pass an array of levels to create a mix of message types.
+	 * For instance,
+	 *     $levelsHandled = [1, 3, 8];
+	 * would result in emergency, alert, and debug messages being logged.
+	 *
+	 * There is no meaning to the order of the values in the array.
+	 * If you put a zero in an array the effect is same as a single int value of 0 which
+	 * turns file logging off.
+	 *
+	 * If you put a single value in an array then only that level will be logged.
+	 * For instance, using
+	 *     $levelsHandled = [8];
+	 * then only 'debug' messages get logged.
+	 *
+	 * For a live site you'll usually want critical (3) or lower to be logged, otherwise
+	 * your log files will fill up very fast.
+	 *
+	 * @var integer|array
+	 */
+	public $fileLevelsHandled = 8;
```

Comment:
Please change the name back as this is definitely a BC break since this requires a change in every application that has been started on the framework so far.